modules/gapi/src/backends/fluid/gfluidcore_simd_sse41.hpp

```diff
@@ -419,7 +436,11 @@ CV_ALWAYS_INLINE void calcRowLinear_8UC_Impl_<3>(uint8_t* dst[],
 #endif
 
                     pix1 = _mm_lddqu_si128(reinterpret_cast<const __m128i*>(&tmp[4 * (chanNum * mapsx[x + 4])]));
+#if defined(__i386__) || defined(_M_IX86)
+                    pix2 = _mm_castpd_si128(_mm_load_sd(reinterpret_cast<const double*>(&tmp[4 * (chanNum * (mapsx[x + 4] + 1))])));
+#else
```

Comment:
Is there any opencv define in this case?

As example, i've found code in `modules\videoio\src\cap_pvapi.cpp`
```
 #if defined(_x64) || defined (__x86_64) || defined (_M_X64)
#define _x64 1
#elif defined(_x86) || defined(__i386) || defined (_M_IX86)
#define _x86 1
#endif
```
where `_x86` as considered as valuable too

Reply:
There is no such definition.
Defining global variable without `CV_` prefix and with `_` is dangerous. It should be avoided, especially in public headers.

OpenCV SIMD wrappers uses this:
- https://github.com/opencv/opencv/blob/4.5.5/modules/core/include/opencv2/core/hal/intrin_sse.hpp#L258-L264
- or more complicated case: https://github.com/opencv/opencv/blob/4.5.5/modules/core/include/opencv2/core/hal/intrin_sse.hpp#L247

Reply:
bad example - my fault
 i considered `_x64` as yet another one  MACRO to check  in current sequence: didn't focus that it declared as custom new MACRO